src/tests/src/solver/infeasible-problem-analysis/test-unfeasible-problem-analyzer.cpp

```diff
@@ -83,7 +87,73 @@ class AnalysisMock: public UnfeasibilityAnalysis
     bool& hasPrinted_;
 };
 
-BOOST_AUTO_TEST_SUITE(unfeasible_problem_analyzer)
+/*!
+ * Creates a 2 problems (feasible and infeasible) with 2 variables linked by 1 constraint:
+ *  - Variable 1 must be greater than 1
+ *  - Variable 2 must be smaller than -1
+ *  - For infeasible problem, constraint enforces that variable 2 is greater than variable 1
+ */
+std::unique_ptr<MPSolver> createProblem(const std::string& constraintName)
+{
+    std::unique_ptr<MPSolver> problem(MPSolver::CreateSolver("GLOP"));
+    const double infinity = problem->infinity();
+    problem->MakeNumVar(1, infinity, "var1");
+    problem->MakeNumVar(-infinity, -1, "var2");
+    auto constraint = problem->MakeRowConstraint(constraintName);
+    constraint->SetBounds(0, infinity);
+    return problem;
+}
```

Comment:
```suggestion
std::tuple<std::unique_ptr<MPSolver>, MPVariable*, MPVariable*> createProblem(const std::string& constraintName)
{
    std::unique_ptr<MPSolver> problem(MPSolver::CreateSolver("GLOP"));
    const double infinity = problem->infinity();
    MPVariable* var1 = problem->MakeNumVar(1, infinity, "var1");
    MPVariable* var2 = problem->MakeNumVar(-infinity, -1, "var2");
    auto constraint = problem->MakeRowConstraint(constraintName);
    constraint->SetBounds(0, infinity);
    return {problem, var1, var2};
}
```
So you don't need `LookupVariableOrNull` afterwards.

Reply:
I replace this code by some other code, simpler, and function **createProblem(...)** is now gone.